app/controllers/employees_controller.rb

```diff
@@ -18,14 +16,38 @@ def new
   def create
     @employee = Employee.new(employee_params)
     if @employee.save
-      redirect_to "/employees"
+      redirect_to '/employees'
     else
       render :new, status: :unprocessable_entity
     end
   end
 
-  private
-    def employee_params
-      params.require(:employee).permit(:email, :name, :position, :employee_number, :private_number)
+  def edit
+    set_employee
+  end
+
+  def update
+    set_employee
+    if @employee.update(employee_params)
+      redirect_to '/employees'
+    else
+      render :edit, status: :unprocessable_entity
     end
   end
+
+  def destroy
+    set_employee
```

Comment:
Instead of calling set employee on all of this lines you can just use 
`before_action :set_employee, only: [:show, :edit, :update, :destroy]` at the start of the controller